### DISCUSSION TITLE
Spree Wordpress Dashboard Integration  & Current Slack Channel

### DISCUSSION THREAD

**Discussion by JessMTermini** [spree/spree#12014] [Q&A]
Hello, I was looking over Spree features and it states very clearly on its features page that some kind of Wordpress integration exists to allow the user to use the wordpress admin to carry out certain tasks for Spree such as adding "to cart from blog posts, display cart, enable checkout, enrich product listings with editorials", with "Spree and WP under one domain".

![Screenshot 2024-03-04 20 48 00](https://github.com/spree/spree/assets/100001251/863e8408-1bc6-495c-9e71-2be16d7e2111)

But after searching through all the developer docs as well as the Spree user documentation, extensions guides and basically everywhere I can't find anything that specifically explains anything at all about these features, what exactly is meant by what is being said on the features page or how to set-up or use the features it is referring to. 

Can someone point me in the right direction on where I can find more information on these features specifically? 

I have also checked the Wordpress public repository and wasn't able to find any Wordpress Plugin for Spree...

-----------------------------------------------

Also I was going to try the Slack channel listed in the current Spree websites navigation, but it points to a google Wayback machine page that no longer exists!! When I see stuff like this it makes me feel like Spree is over promising with filler and fluff content and may not be all it is saying it is cracked up to be, and it truly saddens me and I am really hoping this is not the case. 

Can someone point me to the current slack channel if one currently exists?

Thanks

![Screenshot 2024-03-04 20 49 23](https://github.com/spree/spree/assets/100001251/2e31ab3b-6394-4e62-9bf0-097cb98d360e)
![Screenshot 2024-03-04 20 48 26](https://github.com/spree/spree/assets/100001251/8f497a43-0628-4301-9428-c6d449470ea9)

**Comment by damianlegawiec** [comment]
Hi @JessMTermini thanks for reporting the slack link - it's fixed now!
